src/main/java/programminglife/gui/controller/GuiCreateBookmarkController.java

```diff
@@ -48,7 +48,7 @@ private void initButtons() {
                     Alerts.warning("Center node is not present in graph").show();
                 } else if (inputRadius < 0) {
                     Alerts.warning("Radius can only contain positive integers").show();
-                } else if (txtBookmarkName.getLength() <= 0) {
+                } else if (!txtBookmarkName.getText().matches(".*\\w.*")) {
```

Comment:
I think this matches only things with word boundaries, so something like `~~` wouldn't be matched. I recommend accepting everything that contains a non-whitespace character, so something like `accept if name matches "\\S"`